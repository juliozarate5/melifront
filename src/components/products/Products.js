import React from 'react';
import Product from './Product';
import PropTypes from 'prop-types';

export default function Products({products}) {

    return (
        <div className="row mt-5">
            {
                // eslint-disable-next-line
                products.map((d, i) => 
                    (
                        <Product
                            d={d} 
                            i={i}
                        />
                    )
                )
            }
            </div>
    )
}

Products.propTypes = {
    products: PropTypes.array
}