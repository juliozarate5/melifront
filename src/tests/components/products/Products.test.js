import React from "react";
import { shallow } from "enzyme";
import Products from "../../../components/products/Products";

describe('products tests', () => {

    const products = [];

    test('showing component successful', () => {
        const wrapper = shallow(<Products products={products}/>)
        expect(wrapper).toMatchSnapshot();
    });

});