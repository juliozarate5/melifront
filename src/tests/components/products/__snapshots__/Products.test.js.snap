// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`products tests showing component successful 1`] = `
<div
  className="row mt-5"
/>
`;
